schematools/contrib/django/factories.py

```diff
@@ -108,6 +109,29 @@ def handle_relation(
                 kwargs["relation"] = relation
                 return LooseRelationField, args, kwargs
 
+        if False: #nm_relation:
```

Comment:
Is deze code bewust uitgezet, of is dit een debug-dingetje dat is blijven staan?

Reply:
Vreemd, die had ik lokaal al gewist. Had er niet in gemoeten.